import java.util.Scanner;
public class FirstLastElementArray
{
public static void main(String args[])
{
Scanner sc = new Scanner(System.in);
int n;
System.out.print("Enter number of elements: ");
n = sc.nextInt();
int[] a = new int[n];
System.out.println("Enter array elements:");
for (int i = 0; i < n; i++)    
{
a[i] = sc.nextInt();
}
System.out.println("Array in reverse order:");
for (int i = n - 1; i >= 0; i--)    
{
System.out.print(a[i] + " ");
}
sc.close();

}
}
